src/conventional-commits.ts

```diff
@@ -93,7 +93,12 @@ class PostProcessCommits extends Transform {
     done: Function
   ) {
     chunk.notes.forEach(note => {
-      note.text = note.text.split(/\r?\n/)[0];
+      let text = '';
+      for (const chunk of note.text.split(/\r?\n/)) {
+        if (chunk === '') break;
+        else text += `${chunk} `;
```

Comment:
Hmmm.  I've been known to write fairly lengthy upgrade guides for folks inside of the extended commit message, with full scale examples of old vs new.  I'm a little worried that removing newlines and replacing those with spaces is going to be a less than ideal experience.  

Can I float a different idea?  Typically in releases with Breaking Changes, I will include a list of commits (exactly what we do now) with the links to the corresponding PRs, but I will also have a dedicated "Breaking Changes" section at the very top of the changelog.  There, I like to go into detail on each breaking change, how to mitigate, etc.  Far more prose than we do these days.  What do you think of iterating over breaking changes, taking the extended commit messages, and doing something more prominent at the top of the log?   Example:
https://github.com/googleapis/google-auth-library-nodejs/releases/tag/v2.0.0

Reply:
I think this behavior is what you'd expect from the spec:

```
feat: I am a feature that's a breaking change

BREAKING CHANGE: description line one
description line two
description line three

additional content.
more additional content.
```

* the detailed information about the breaking change is in its own section of the CHANGELOG, under breaking changes (_the list of commits will have only the one line description._).
* `additional content` and `more additional content` will not be in the CHANGELOG.

Here's example output from the unit tests:

```md
## v1.0.0 (1665-10-10)

### ⚠ BREAKING CHANGES

* we were on Node 6 second line third line

### Features

* awesome feature ([abc678](https://www.github.com/bcoe/release-please/commit/abc678))
```

I think if we do decide we want to do anything differently, we should  capture it in the spec on conventionalcommits.org, so that all the parsers behave the same.